dev-util/scons/scons-3.0.5-r1.ebuild

```diff
@@ -67,6 +67,8 @@ src_prepare() {
 python_test() {
 	# set variable from escons() of scons-util.eclass to make env-passthrough patch work within test env
 	local -x GENTOO_SCONS_ENV_PASSTHROUGH=1
+	# unset some env variables to pass appropriate tests
+	unset CC CFLAGS CXXFLAGS LDFLAGS
```

Comment:
`CXX` maybe too, for completeness? Also `CPPFLAGS`. I'd generally go for all the things that are handled by our patch.